app/models/placement_group.rb

```diff
@@ -0,0 +1,81 @@
+class PlacementGroup < ApplicationRecord
+  acts_as_miq_taggable
+  include SupportsFeatureMixin
+  include NewWithTypeStiMixin
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
```

Comment:
Minor but can you add a newline before the declaration of the associations?
```suggestion

  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
```